ooni/settings.py

```diff
@@ -156,6 +156,9 @@
 
 # These are the the embedded settings
 _SETTINGS_INI = os.path.join(OONIPROBE_ROOT, 'settings.ini')
+# This can be overridden by an environment variable for end to end testing
+if 'OONIPROBE_SETTINGS' in os.environ:
+    _SETTINGS_INI = os.environ['OONIPROBE_SETTINGS']
```

Comment:
`os.environ.get('OONIPROBE_SETTINGS', os.path.join(OONIPROBE_ROOT, 'settings.ini'))` is a bit more pythonic :)